src/custom/state/swap/extension.ts

```diff
@@ -114,9 +120,11 @@ export function useTradeExactInWithFee({
 
   // Original Uni trade hook
   const inTrade = useTradeExactIn(feeAdjustedAmount, outputCurrency ?? undefined)
+  const originalTrade = useTradeExactIn(parsedAmount, outputCurrency ?? undefined)
```

Comment:
i dont like this whole thing but this line here i to get the original, non-fee adjusted, `outputAmount` to calculate fees when running a `sell` order